common/script/ops/hourglassPurchase.js

```diff
@@ -3,53 +3,58 @@ import i18n from '../i18n';
 import _ from 'lodash';
 import splitWhitespace from '../libs/splitWhitespace';
 import pickDeep from '../libs/pickDeep';
+import {
+  BadRequest,
+  NotAuthorized,
+} from '../libs/errors';
+
+module.exports = function purchaseHourglass (user, req = {}, analytics) {
+  let key = _.get(req, 'body.key');
```

Comment:
same thing as purchase here, it should be req.parameters https://github.com/HabitRPG/habitrpg/blob/develop/website/src/routes/api-v2/swagger.js#L170